synapse/rest/client/v1/room.py

```diff
@@ -438,6 +438,7 @@ async def on_POST(self, request, room_id):
         prev_state_ids = list(prev_state_map.values())
```

Comment:
I've just realised that we should probably not be adding this class to `v1/room.py`, but instead somewhere in `v2/`. If only someone had reviewed the PR that added this endpoint.................. :facepalm:

Reply:
I can switch it over in another PR 😁

Reply:
https://github.com/matrix-org/synapse/pull/10576 ⏩